core/types/transaction_signing.go

```diff
@@ -168,6 +168,9 @@ type Signer interface {
 
 	// Equal returns true if the given signer is the same as the receiver.
 	Equal(Signer) bool
+
+	// Unwrap returns unwrapped signer implementation
+	Unwrap() interface{}
```

Comment:
nit: The method might be restricted to return a Signer:
```suggestion
	Unwrap() Signer
```